debug/index.js

```diff
@@ -85,7 +85,10 @@ var geocoder = new MapboxGeocoder({
         .then(response => response.json())
     }
   },
-  mapboxgl: mapboxgl
+  mapboxgl: mapboxgl,
+  reverseGeocode: true,
+  flipCoordinates: true,
```

Comment:
Do we have flip-coordinates on by default in v6? Let's discuss

Reply:
No, it's debug file for testing purposes. flip coordinates by default set to false
https://github.com/mapbox/mapbox-gl-geocoder/blob/d49005774eff3bd0fd3edc8b4e521b4b9a5cc9bb/lib/index.js#L138